core/src/main/java/com/linecorp/armeria/common/logging/AbstractSanitizerBase.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2017 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common.logging;
+import java.util.function.Function;
+
+/**
+ * Base class.
+ */
+public class AbstractSanitizerBase implements Function<Object, Object> {
```

Comment:
I guess we don't really need this type. Could you remove this class and just call `toString()` in `RegexBasedSanitizer`?

Reply:
I removed it.